streams-tests/jvm/src/test/scala/zio/stream/compression/TestData.scala

```diff
@@ -86,7 +86,7 @@ object TestData {
   val shortText: Array[Byte]          = "abcdefg1234567890".getBytes
   val otherShortText: Array[Byte]     = "AXXX\u0000XXXA".getBytes
   val longText: Array[Byte]           = Array.fill(1000)(shortText).flatten
-  val `1K`                            = 1024
+  val `1K`                            = 128
```

Comment:
The name is wrong now

Reply:
I wanted to see if this actually helped before renaming it but yes. 😃